e2e/pages/caseView.page.js

```diff
@@ -38,13 +38,13 @@ module.exports = {
   async goToNewActions(actionSelected) {
     const currentUrl = await I.grabCurrentUrl();
     await I.retryUntilExists(async () => {
-      if(await I.waitForSelector(this.actionsDropdown, 60) != null) {
+      if(await I.waitForSelector(this.actionsDropdown, 30) != null) {
         await I.scrollToElement(this.actionsDropdown);
         I.selectOption(this.actionsDropdown, actionSelected);
         I.click(this.goButton);
       } else {
         const newUrl = await I.grabCurrentUrl();
-        if(newUrl === currentUrl){
+        if(newUrl === currentUrl || !newUrl.includes('http')){
```

Comment:
guess this is that IE issue you were talking about

Reply:
yeah, that's it, hopefully it'll help - I've not actually been able to hit the same ExUI flakey issue to prove it, but it's not caused any failures either 🙂